demo-shell/src/app/components/app-layout/app-layout.component.scss

```diff
@@ -14,6 +14,10 @@
         }
     }
 
+    adf-file-uploading-dialog {
+        z-index: 1100;
```

Comment:
do we need this in the demo shell? does that mean every application needs to have this? if not, then let's have this as part of the uploading dialog styles